requirements.txt

```diff
@@ -0,0 +1,4 @@
+scapy
+flask
+grpcio
+protobuf
```

Comment:
perhaps nail these to the latest known working versions?

Reply:
I think, if it has some known limitation for a specific version (e.g., known minimum), we can specify that. If not, I think we shouldn't specify versions, as we don't actually want to force to use that specific version.